modules/apps/object/object-service/src/main/java/com/liferay/object/internal/notification/term/contributor/ObjectDefinitionNotificationTermEvaluator.java

```diff
@@ -54,15 +63,52 @@ public String evaluate(Context context, Object object, String termName)
 
 		Map<String, Object> termValues = (Map<String, Object>)object;
 
-		if (termName.contains("_CREATOR")) {
-			if (context.equals(Context.RECIPIENT)) {
-				return String.valueOf(termValues.get("creator"));
+		User user = null;
+
+		String prefix = StringUtil.toUpperCase(
+			_objectDefinition.getShortName());
+
+		Set<String> creatorTermNames = Collections.unmodifiableSet(
+			SetUtil.fromArray(
+				"[%" + prefix + "_CREATOR%]",
+				"[%" + prefix + "_CREATOR_EMAIL%]",
+				"[%" + prefix + "_CREATOR_FIRSTNAME%]",
+				"[%" + prefix + "_CREATOR_ID%]",
+				"[%" + prefix + "_CREATOR_LASTNAME%]",
+				"[%" + prefix + "_CREATOR_MIDDLENAME%]",
+				"[%" + prefix + "_CREATOR_PREFIX%]",
+				"[%" + prefix + "_CREATOR_SUFFIX%]"));
+
+		if (creatorTermNames.contains(termName)) {
+			if (!FeatureFlagManagerUtil.isEnabled("LPS-171625")) {
```

Comment:
@p-albuquerque 

it won't work after removing the FF, here we need to make sure that the term is `[%" + prefix + "_CREATOR%]` to proceed execute the old statement.

Reply:
I believe it will work in the next commit